tools/twix/src/panels/manual_camera_calibration.rs

```diff
@@ -39,7 +39,7 @@ impl Panel for ManualCalibrationPanel {
             let path = CAMERA_KEY_BASE.to_owned() + name.to_lowercase().as_str() + ROTATIONS;
 
             let (update_notify_sender, update_notify_receiver) = mpsc::channel(1);
-            let value_buffer = subscribe(nao.clone(), &path, update_notify_sender);
+            let value_buffer = subscribe(nao.clone(), &path, update_notify_sender).unwrap();
```

Comment:
I was wondering if this could fail if there is no connection to the nao yet. While testing I moved some sliders and reconnected some times using the connected check. This crashed consistently not at this line but here 
`thread 'tokio-runtime-worker' panicked at 'called `Result::unwrap()` on an `Err` value: SendError { .. }', hulk/crates/communication/src/client/communication.rs:146:14`

Reply:
But just checked, this problem also exists on `main` :sweat_smile:

Reply:
Yes, there are still a bunch of `unwrap`s in the communication client library that should be replaced with more graceful error handling.